gnosis/cowswap
# 2538/Claim page crash on reload fix

# Summary

Closes #2538 

Moving custom components to mod file to avoid circular dependencies/lazy loading issues

  # To Test

1. Open the claim page
2. Refresh it
* It should NOT crash and the page should load fine

  # Background

*Optional: Give background information for changes you've made, that might be difficult to explain via comments*
